plugins/catalog-backend-module-ldap/src/ldap/types.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { GroupEntity, UserEntity } from '@backstage/catalog-model';
+import { SearchEntry } from 'ldapjs';
+import { LdapVendor } from './vendors';
+import { GroupConfig, UserConfig } from './config';
+
+/**
+ * Customize the ingested User entity
+ *
+ * @param vendor
+ * @param config
+ * @param user
```

Comment:
you can just leave out these if you aren't adding any text to them. But I think it could be worth doing so, and also add a `@returns` that explains what undefined means